ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,155 +1,169 @@
 {# eos - VLAN Interfaces #}
-{% if vlan_interfaces is defined and vlan_interfaces is not none %}
+{% if vlan_interfaces is arista.avd.defined %}
 {%     for vlan_interface in vlan_interfaces | arista.avd.natural_sort %}
 !
 interface {{ vlan_interface }}
-{%         if vlan_interfaces[vlan_interface].description is defined and vlan_interfaces[vlan_interface].description is not none %}
+{%         if vlan_interfaces[vlan_interface].description is arista.avd.defined %}
    description {{ vlan_interfaces[vlan_interface].description }}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].shutdown is defined and vlan_interfaces[vlan_interface].shutdown == true %}
+{%         if vlan_interfaces[vlan_interface].shutdown is arista.avd.defined(true) %}
    shutdown
-{%         elif vlan_interfaces[vlan_interface].shutdown is defined and vlan_interfaces[vlan_interface].shutdown == false %}
+{%         elif vlan_interfaces[vlan_interface].shutdown is arista.avd.defined(false) %}
    no shutdown
 {%         endif %}
 {%         if vlan_interfaces[vlan_interface].mtu is defined and vlan_interfaces[vlan_interface].mtu != 1500 %}
```

Comment:
The new style is to configure MTU no matter what the value is. Defaults could have been changed to something else than 1500. Also use arista.avd.defined filter.